Client/Frontend/Browser/BrowserViewController/BrowserViewController+ProductNotification.swift

```diff
@@ -183,4 +268,36 @@ extension BrowserViewController {
             }
         }
     }
+    
+    func shareTrackersAndAdsWithEmail(_ count: Int) {
+        benchmarkNotificationPresented = false
```

Comment:
What is this flag used for?

Reply:
this is is the flag used for guarding when a type of notification is already on screen another type will not replace until previous one is dismissed by user.